src/ApplicationLogs/LogReader.cs

```diff
@@ -96,7 +96,7 @@ public static JObject TxLogToJson(Blockchain.ApplicationExecuted appExec)
             }
             catch (Exception ex)
             {
-                trigger["exception"] = ex.Message;
+                trigger["stackexception"] = ex.Message;
```

Comment:
Current behavior was introduced in 81eb9b78393a080075f53afbe7ae7d915693e712, pretty much accidentally (hi, monorepo). See #854 for NeoGo behavior wrt this (which is closer to the older one). I don't like how we randomly change these things, it's a public API after all.

Reply:
We need to ensure application logs and test invocations are treated the same wrt this.

Reply:
Seems from https://github.com/neo-project/neo-modules/pull/696, how about revert to the older one?

Reply:
> Seems from #696, how about revert to the older one as `stack`?

i vote this.